distribution/docs/src/main/resources/content/_integrating/_endpoints/ftp-endpoint.adoc

```diff
@@ -2,49 +2,40 @@
 :type: endpoint
 :status: published
 :link: _ftp_endpoint
+:operations: ingest
+:url: \ftp://<FQDN>:8021/
 :summary: Ingests files directly into the ${branding} Catalog using the FTP protocol.
 :implements: https://tools.ietf.org/html/rfc959[FTP] {external-link}
 
-The FTP Endpoint provides a method for ingesting files directly into the ${branding} Catalog using the FTP protocol.
-The files sent over FTP are not first written to the file system, like the Directory Monitor, but instead the FTP stream of the file is ingested directly into the ${branding} catalog, thus avoiding extra I/O overhead.
-
-===== Installing the FTP Endpoint
-
-The FTP Endpoint is not installed by default with a standard installation.
+== {title}
 
-To install:
-
-. Navigate to the *${admin-console}*.
-. Select the *System* tab.
-. Select the *Features* tab.
-. Install the *catalog-core-ftp* feature.
+The FTP Endpoint provides a method for ingesting files directly into the ${branding} Catalog using the FTP protocol.
 
-===== Configuring the FTP Endpoint
+The FTP endpoint can be accessed from any FTP client of choice.
+Some common clients are FileZilla, PuTTY, or the FTP client provided in the terminal.
+The default port number is *8021*. If FTPS is enabled with 2-way TLS, a client that supports client authentication is required.
 
-Once installed, the configurable properties for the FTP Endpoint are accessed from the *FTP Endpoint* Configuration:
+Custom Ftplets can be implemented by extending the `DefaultFtplet` class provided by Apache FTP Server.
+Doing this will allow custom handling of various FTP commands by overriding the methods of the `DefaultFtplet`.
+Refer to https://mina.apache.org/ftpserver-project/ftplet.html {external-link} for available methods that can be overridden.
+After creating a custom Ftplet, it needs to be added to the FTP server’s Ftplets before the server is started. Any Ftplets that are registered to the FTP server will execute the FTP command in the order that they were registered.
 
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-catalog}* application.
-. Select the *FTP Enpoint*.
+.Operations
+[cols="2,1m,2,5m", options="header"]
+|===
 
-See <<{reference-prefix}ddf.catalog.ftp.FtpServerManager,FTP Endpoint configurations>> for all possible configurations.
+|Operation
+|FTP Request Type
+|Details
+|Example URL
 
-===== Using FTP Endpoint
+|ingest
+|PUT
+|
+|{url}
 
-.FTP Endpoint URL
-----
-ftp://<FQDN>:8021/
-----
+|===
```

Comment:
✏️ I know you're not fleshing out the content of these tables yet, but don't you need a row for all these possible operations? `PUT`, `MPUT DELE`, `RETR`, `RMD`, `APPE`, `RNTO`, `STOU`, and `SITE`

Reply:
I was assuming so. I'm going to tag the next issue so this doesn't get overlooked: https://github.com/codice/ddf/issues/4374